phpBB/phpbb/cron/task/text_reparser/reparser.php

```diff
@@ -99,7 +99,7 @@ public function is_runnable()
 			$this->reparser_manager->get_resume_data($this->reparser_name);
 		}
 
-		if (empty($this->resume_data['range-max']) || $this->resume_data['range-max'] >= $this->resume_data['range-min'])
+		if (!isset($this->resume_data['range-max']) || $this->resume_data['range-max'] >= $this->resume_data['range-min'])
```

Comment:
I think this might need to be this:

```php
if (!empty($this->resume_data['range-max']) && $this->resume_data['range-max'] >= $this->resume_data['range-min'])
```

IOW, only run the task if `range-max` is positive (non-zero) and the range is valid.

@Elsensee does that make sense to you? I can't run it at the moment to verify it. That may be the only change required.